client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -1,5 +1,43 @@
-const ZoneCreator = () => {
-	return null;
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import page from 'page';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+const ZoneCreator = ( { siteSlug, translate } ) => {
+	const handleGoBack = () => {
+		page( `/extensions/zoninator/${ siteSlug }` );
+	};
+
+	return (
+		<div>
+			<HeaderCake onClick={ handleGoBack }>
+				{ translate( 'Add a zone' ) }
+			</HeaderCake>
+		</div>
```

Comment:
We could but there's going to be more content in there eventually. In fact, these components are pretty much re-written in #16254 and #16497.